mariadb 0.3.1

prydonius opened the pull request:
Updates MariaDB image version to 10.1.17-r1 including fixes for the
OpenSSL security update
(https://www.openssl.org/news/secadv/20160922.txt)

gsanchezu commented:
Hi @kmala, is there anything required to merge this?

bacongobbler commented:
nope! All is done. Just verifying that specific version of jenkins exposes the JNLP port on 50000 on my end, then we should be good to merge.

bacongobbler commented:
LGTM after viewing the history of this image:

```
9a950ac7d681        3 months ago        /bin/sh -c #(nop) EXPOSE 50000/tcp
2e7b98150a55        3 months ago        /bin/sh -c #(nop) EXPOSE 8080/tcp
```

Commit:
mariadb 0.3.1

Updates MariaDB image version to 10.1.17-r1 including fixes for the
OpenSSL security update
(https://www.openssl.org/news/secadv/20160922.txt)

prydonius commented on stable/mariadb/Chart.yaml:
```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 0.3.0
+version: 0.3.1
```
It would be good to discuss how we want to version charts at some point, some general guidelines would be good.

viglesiasce commented:
LGTM.
